docs/api/remote.md

```diff
@@ -1,5 +1,7 @@
 # remote
 
+> Communicate between the renderer process and the main process.
```

Comment:
`remote` is mainly about using modules in the main process from the renderer process,